upstream-community-operators/kiali/kiali.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,202 @@
+    mediatype: image/png
+  keywords: ['service-mesh', 'observability', 'monitoring']
+  version: 0.1.0
+  #replaces: kiali-operator.v0.0.0
```

Comment:
Please remove this comment for `replaces` as it is unnecessary, since this is the first CSV

Reply:
removed.